website/content/docs/api-gateway/index.mdx

```diff
@@ -64,22 +58,33 @@ Your datacenter must meet the following requirements prior to configuring the Co
    <CodeBlockConfig>
 
    ```shell-session
-   $ helm install consul hashicorp/consul --version 0.40.0 --values values.yaml
+   $ helm install consul hashicorp/consul --version 0.41.0 --values values.yaml
    ```
 
    </CodeBlockConfig>
 
 ## Usage
 
 1. Verify that the [requirements](#requirements) have been met.
-1. Verify that the Consul API Gateway software has been installed and applied (see [Installation](#installation)).
-1. Configure the artifacts described in [Configuration](#configuration).
+1. Verify that the Consul API Gateway CRDs and controller have been installed and applied (see [Installation](#installation)).
+1. Configure the artifacts described below in [Configuration](#configuration). If the following config is added to the `values.yaml` Helm config, the GatewayClassConfig and GatewayClass will be created automatically, and only the Gateway, Listeners and Routes will need to be configured manually. The [`serviceType`](https://www.consul.io/docs/k8s/helm#v-apigateway-managedgatewayclass-servicetype) for a managed GatewayClass will default to `LoadBalancer`, which is appropriate for most deployments to managed Kubernetes cloud offerings (EKS, GKE, AKS) - other deployments, such as to a [kind](https://kind.sigs.k8s.io/) cluster, may require specifying `NodePort` or `ClusterIP` instead.
+
+   <CodeBlockConfig hideClipboard filename="values.yaml">
+
+   ```yaml
+   apiGateway:
+     managedGatewayClass:
+       enabled: true
+   ```
+
+   </CodeBlockConfig>
+
 1. Issue the `kubectl apply` command to implement the configurations, e.g.:
 
    <CodeBlockConfig>
 
    ```shell-session
-   $ kubectl apply --values gateway-configuration.yaml
+   $ kubectl apply -f gateway.yaml routes.yaml
```

Comment:
Since we have no reference to something like a `gateway.yaml` or `routes.yaml` file here, should we add an example?

Reply:
If you read closely, the sections below in Configuration do instruct the user to add the example content to a `gateway.yaml` or `routes.yaml` file - it's not really obvious though, but I couldn't think of a way to improve it that wasn't a more significant restructuring.